pkg/provider/conjur/provider_test.go

```diff
@@ -495,7 +495,7 @@ func TestGetCA(t *testing.T) {
 			args: args{
 				store: makeStoreWithCA("secret", ""),
 				kube: clientfake.NewClientBuilder().
-					WithObjects(makeFakeCASource("secret", certData)).Build(),
+					WithObjects(makeFakeCASource("secret", certDataEncoded)).Build(),
```

Comment:
Why did we need to update this test? 🤔

Reply:
Ah. There was some confusion when I moved around the base64 decoding code. Maybe I either didn't move it right or I moved it right and previously it wasn't working right or it wasn't using the right value and now it does. I will need to carefully investigate that. 😁

Reply:
Set the decode to auto, this should be fixed now.